src/openforms/contrib/digid_eherkenning/utils.py

```diff
@@ -7,11 +7,13 @@ def get_digid_logo(request) -> Dict[str, str]:
     return {
         "image_src": request.build_absolute_uri(static("img/digid-46x46.png")),
         "href": "https://www.digid.nl/",
+        "appearance": "dark",
     }
 
 
 def get_eherkenning_logo(request) -> Dict[str, str]:
     return {
         "image_src": request.build_absolute_uri(static("img/eherkenning.png")),
         "href": "https://www.eherkenning.nl/",
+        "appearance": "light",
     }
```

Comment:
this should use the `LoginLogo` data class instead of returning a raw dict

Reply:
I'm wondering if `utils.py` should be deleted and the functions implemented directly in `openforms/authentication/contrib/digid/plugin.py` and `openforms/authentication/contrib/digid_eherkenning_oidc/plugin.py`. It seems that nothing is gained (except a layer of indirection) by exporting them to a separate file.